renderer/configuration/robotouille_config.json

```diff
@@ -10,6 +10,15 @@
         }
     },
 
+    "customer": {
+        "robot": {
```

Comment:
may as well remove the "robot" key for the time being until we have different assets to make this hardcoding more evident.

Reply:
changed it to girl so that the choose_character_image_name() function can take in a name